app/uk/gov/hmrc/plasticpackagingtax/registration/models/registration/OrganisationDetails.scala

```diff
@@ -50,6 +50,12 @@ case class OrganisationDetails(
     else if (businessPartnerIdPresent()) TaskStatus.Completed
     else TaskStatus.InProgress
 
+  def businessVerificationFailureStatus(): Boolean =
```

Comment:
NIT:  Would this method name be better?  `businessVerificationFailed`
And it can be a `val` rather than a `def` (as can the other existing `def`s for that matter)